Plugin: setu_customization

**插件名称：**

setu_customization

**插件功能：**

可动态管理API并带网页浏览的setu插件

**PyPI 项目名：**

nonebot-plugin-setu-customization

**插件 import 包名：**

nonebot_plugin_setu_customization

**插件项目仓库/主页链接：**

https://github.com/nikissXI/nonebot_plugins/tree/main/nonebot_plugin_setu_customization

**标签：**

[{"label":"色图","color":"#e9ea52"}]

# 📃 商店发布检查结果

> Plugin: setu_customization

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot_plugin_setu_customization 的信息如下：
    name         : nonebot-plugin-setu-customization 
     version      : 0.1.3                             
     description  : Nonebot2 可动态管理API并带网页浏览的setu插件    
    
    dependencies
     - beautifulsoup4 &gt;=4.0.0,&lt;5.0.0
     - httpx &gt;=0.23.0,&lt;0.24.0
     - httpx-socks &gt;=0.7.3,&lt;0.8.0
     - nonebot2 &gt;=2.0.0-rc.1,&lt;3.0.0
     - Pillow &gt;=9.0.0,&lt;10.0.0
     - starlette &gt;=0.19.0,&lt;0.20.0
     - ujson &gt;=5.0.0,&lt;6.0.0
插件 nonebot_plugin_setu_customization 加载出错：
01-03 06:07:18 [SUCCESS] nonebot | NoneBot is initializing...
01-03 06:07:18 [INFO] nonebot | Current Env: prod
01-03 06:07:18 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_setu_customization&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_setu_customization&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_setu_customization/__init__.py&quot;, line 9, in &lt;module&gt;
    from nonebot.adapters.onebot.v11 import Bot, GroupMessageEvent, MessageEvent
ModuleNotFoundError: No module named &#x27;nonebot.adapters.onebot&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: 色图-#e9ea52。</li><li>✅ 项目 <a href="https://github.com/nikissXI/nonebot_plugins/tree/main/nonebot_plugin_setu_customization">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot_plugin_setu_customization/">nonebot_plugin_setu_customization</a> 已发布至 PyPI。</li></code></pre></details>

---

💪 Powered by NoneBot2 Publish Bot
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -3847,5 +3847,20 @@
       }
     ],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_setu_customization",
+    "project_link": "nonebot-plugin-setu-customization",
+    "name": "setu_customization",
+    "desc": "可动态管理API并带网页浏览的setu插件",
+    "author": "nikissXI",
+    "homepage": "https://github.com/nikissXI/nonebot_plugins/tree/main/nonebot_plugin_setu_customization",
+    "tags": [
+      {
+        "label": "色图",
+        "color": "#e9ea52"
+      }
+    ],
+    "is_official": false
   }
 ]

